config/config.go

```diff
@@ -25,9 +25,11 @@ type Config struct {
 	IdentityPublicKey string `yaml:"identity-public-key"`
 	IdentityLocation  string `yaml:"identity-location"`
 	// The identity API is optional
-	IdentityAPIURL      string `yaml:"identity-api-url"`
-	IdentityAPIUsername string `yaml:"identity-api-username"`
-	IdentityAPIPassword string `yaml:"identity-api-password"`
+	IdentityAPIURL        string `yaml:"identity-api-url"`
+	IdentityAPIUsername   string `yaml:"identity-api-username"`
+	IdentityAPIPassword   string `yaml:"identity-api-password"`
+	IdentityAPIPrivateKey string `yaml:"identity-api-private-key"`
```

Comment:
I think this would be more appropriate to just call this PrivateKey, as it will be used for agent authentication to any third party discharge service. Even better if it was declared as a bakery.KeyPair, although that will require more logic in this package.